tests/baselines/reference/tsbuild/watchMode/programUpdates/incremental-updates-in-verbose-mode.js

```diff
@@ -705,9 +705,9 @@ export declare function someFn(): void;
     },
     "semanticDiagnosticsPerFile": [
       "../../../../../a/lib/lib.d.ts",
-      "../core/anothermodule.ts",
-      "../core/index.ts",
-      "../logic/index.ts",
+      "../core/anothermodule.d.ts",
+      "../core/index.d.ts",
+      "../logic/index.d.ts",
```

Comment:
What do all these .ts → .d.ts changes indicate? Do they have an effect on how the user sees diagnostics reported during watch mode or in the editor, or is it purely an internal representation?

Reply:
These are only .tsbuildinfo changes .. no changes to error reporting